recipes/qt.py/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "Qt.py" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "209fd4909241f08983e9cdf8768341b7f56adcc37cc4abacd0bca6ad8620479f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  requires:
+    - PySide2
```

Comment:
this should be `pyside2`

Reply:
Hm, is PySide really available for Python > 3.4?

Reply:
PySide2 was added as a feedstock just an hour ago or so. It should work much better I think. But it wasn't found on all platforms. Trying PySide now.

Reply:
I didn't mean `pyside`. I meant the package name should be lowercase. `pyside2` instead of `PySide2`

Reply:
Doesn't seem like PySide worked either: https://travis-ci.org/conda-forge/staged-recipes/builds/268999200#L278

Reply:
Trying PyQt4 then...

Reply:
Oh! I see now. It was case sensitive!

Reply:
Please see my comment here, https://github.com/conda-forge/staged-recipes/pull/3832#discussion_r135422099

Reply:
Yeah, now it works. Thanks @isuruf 😄